api/RTCPeerConnection.json

```
@@ -111,7 +111,7 @@
             "chrome": [
               {
                 "version_added": "55",
-                "notes": "Before Chrome 63 the default value for the <code>RTCConfiguration.rtcpMuxPolicy</code> parameter was <code>&quot;negotiate&quot;</code>"
+                "notes": "In Chrome 63, the default value for the <code>configuration.rtcpMuxPolicy</code> parameter changed from <code>&quot;negotiate&quot;</code> to <code>&quot;required&quot;</code>."
```

Yeah, I like the idea of making the note standalone better. A couple of adjustments to suggest anyway:

- Avoiding "In version X" when we mean "In several versions of a range starting or ending with X" since "In X" might be interpreted as applying to X and only X.
- Present tense, a style convention I like (kinda related to above, but also older releases aren't usualy changed retroactively so it's a little weird to talk about the behavior in the past tense, though this is less weird the older a release is)

```suggestion
                "notes": "Before Chrome 63, the default value for the <code>configuration.rtcpMuxPolicy</code> parameter is <code>&quot;negotiate&quot;</code> instead of <code>&quot;required&quot;</code>."
```

---

+1